src/navigators/PopupWindow.ts

```diff
@@ -59,7 +59,7 @@ export class PopupWindow extends AbstractChildWindow {
 
         const popupClosedInterval = setInterval(() => {
             if (!this._window || this._window.closed) {
-                void this._abort.raise(new Error("Popup closed by user"));
```

Comment:
for the use-case where we have a window this raise makes still sense. Why not change the condition to?
`if (this._window && this._window.closed) {...`

Reply:
Both cases have this._window non-null unless the window.open() call in the constructor returned null (likely because of a popup blocker). It is not possible to distinguish if this._window.closed is true because the popup was closed by the user or because the url passed to super.navigate() below has Cross-Origin-Opener-Policy: same-origin.

I do see that this will repeatedly log the message until the dispose handler is called.

It also looks like the closePopupWindowAfterInSeconds timeout is going to throw an error if this._window.closed is true, when really it is the return value of window.open() that should be checked.

Reply:
I fixed the repeated log message.